interview-questions.md

```diff
@@ -3,44 +3,46 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: the foreign key would be part of the sightings model, and the foreign key would be called animal_id.
 
-  Researched answer:
+  Researched answer: naming conventions for a foreign key the target table name, followed by the source table name. The table containing the foreign key is called the child table, and the table containing the candidate key is called the referenced or parent table.
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: the routes that are creating, updating, destroying and showing all need to have params passed through them. This is so the user knows exactly what they are accessing and are creating.
```

Comment:
I dont believe you need a param to create since the id is populated upon creation?